config/kustomization.yaml.nope

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 The Sigstore Authors.
```

Comment:
why do we need this file ?

Reply:
haha, got added to commit by mistake, good catch :)